packages/docz-tools/src/gatsby-theme-docz/components/Navigation/LevelThreeLink.tsx

```diff
@@ -0,0 +1,34 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui";
+import { Link } from "docz";
+import { Icon } from "@jobber/components/Icon";
+import { levelThreeLink } from "./styles";
+
+interface LevelThreeLinkProps {
+  readonly title: string;
+  readonly slug: string;
+}
+
+export function LevelThreeLink({ title, slug }: LevelThreeLinkProps) {
+  const link = `#${slug}`;
+  const currentHash = getCurrentHash();
```

Comment:
If they're using react router it may have a hook for this?

Reply:
This is copied from docz